src/cljs/harja/tiedot/urakka/toteumat/maarien_toteumat.cljs

```diff
@@ -199,12 +200,12 @@
   (cond
     (= tyyppi :akillinen-hoitotyo) ;; Äkillinen hoitotyö tässä tarkoittaa, että on valittu käyttöliittymässä "Äkillinen hoitotyö, vahingon korjaus, rahavaraus".
     (some (fn [toimenpide]
-            (when (= "4 LIIKENTEEN VARMISTAMINEN ERIKOISTILANTEESSA" (:otsikko toimenpide))
+            (when (str/includes? (:otsikko toimenpide) "LIIKENTEEN VARMISTAMINEN ERIKOISTILANTEESSA")
               toimenpide))
           (get-in app [:toimenpiteet]))
     (= tyyppi :lisatyo)
     (some (fn [toimenpide]
-            (when (= "7.0 LISÄTYÖT" (:otsikko toimenpide))
+            (when (str/includes? (:otsikko toimenpide) "LISÄTYÖT")
```

Comment:
Tässä tapahtuu NPE jos (:otsikko toimenpide) on nil - jos se on ei-toivottua, voidaan esim kääriä wheniin tai tehdä when-ehto (and (:otsikko toimenpide) (str/includes...)

Reply:
Lisätty tarkistus